benchmark/build.gradle

```diff
@@ -23,6 +22,23 @@ dependencies {
     testImplementation("org.testng:testng:${testng_version}") {
         exclude group: "junit", module: "junit"
     }
+
+    ProcessBuilder pb = new ProcessBuilder("nvidia-smi", "-L")
+    def hasGPU = false;
+    try {
+        Process process = pb.start()
+        hasGPU = process.waitFor() == 0
+    } catch (IOException ignore) {
+    }
+
+    if (hasGPU) {
+        runtimeOnly("ai.djl.onnxruntime:onnxruntime-engine") {
+            exclude group: "com.microsoft.onnxruntime", module: "onnxruntime"
+        }
+        runtimeOnly "com.microsoft.onnxruntime:onnxruntime_gpu:${onnxruntime_version}"
```

Comment:
I don't think we have onnxruntime_version defined in gradle.properties in this repo